.travis.yml

```diff
@@ -1,9 +1,14 @@
 sudo: false
 language: cpp
 compiler:
-- clang
+  - clang
 os:
-- osx
+  - osx
+before_install:
+  - wget https://raw.githubusercontent.com/google/styleguide/gh-pages/cpplint/cpplint.py
```

Comment:
No confusion!
```cpp
some_object.BigAndExpensiveMethod()
some_object.simple_method_or_just_a_getter()
some_struct.element_of_the_struct
```
UpperCamelCase for functions is quite common, although it took me a lot to get used to it lol

We can move the discussion to Slack!